plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveUtil.java

```diff
@@ -749,11 +749,9 @@ public static List<HiveType> getColumnTypes(Map<String, String> schema)
     public static OrcWriterOptions getOrcWriterOptions(Map<String, String> schema, OrcWriterOptions orcWriterOptions)
     {
         if (schema.containsKey(ORC_BLOOM_FILTER_COLUMNS_KEY)) {
-            if (!schema.containsKey(ORC_BLOOM_FILTER_FPP_KEY)) {
-                throw new TrinoException(HIVE_INVALID_METADATA, "FPP for bloom filter is missing");
-            }
             try {
-                double fpp = parseDouble(schema.get(ORC_BLOOM_FILTER_FPP_KEY));
+                // use default fpp DEFAULT_BLOOM_FILTER_FPP if fpp key does not exist in table metadata
```

Comment:
When does this situation happens, if Trino creates a hive table it would populate this property and if hive creates it out - then it would still populate this property right ?

Reply:
Already answered !!